src/static/helpers/slasHelper.ts

```diff
@@ -13,8 +13,8 @@ import {
   ShopperLogin,
   ShopperLoginPathParameters,
   ShopperLoginQueryParameters,
-  TokenResponse,
-} from '../../lib/shopperLogin';
+} from '../../lib/shopperLogin/apis';
+import {TokenResponse} from '../../lib/shopperLogin/models';
```

Comment:
We went over this in the meet earlier but could you restate why this change in particular is needed or how the original version resulted in a conflict in the rolled up typescript?